pom.xml

```diff
@@ -126,6 +126,17 @@
     <jakarta-client-webservices.version>2.0.0.Beta3</jakarta-client-webservices.version>
     <org.littleshoot.littleproxy.version>1.1.2</org.littleshoot.littleproxy.version>
     <gnu.getopt.version>1.0.13</gnu.getopt.version>
+    <!--Add wildfly integration tests dependency version below-->
+    <jakarta.json-api.version>1.1.6</jakarta.json-api.version>
+    <jboss.ejb3.ext-api.version>2.3.0.Final</jboss.ejb3.ext-api.version>
+    <remoting.jmx.version>3.0.4.Final</remoting.jmx.version>
+    <jboss.remoting.version>jboss-remoting</jboss.remoting.version>
```

Comment:
this line looks incorrect and it is ignored anyway as it is overridden by just the next line

Reply:
Fixed. Thanks for review @jbliznak